CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixes 
 
 - Ensure the current GPU frame rate is always reported for concurrent transaction profiling metrics (#2929)
+- Improved performance serializing profiling data (#2863)
```

Comment:
`m`: I think that needs to move up a bit.

Reply:
Are you looking at ordering by PR number? I haven't really cared about it too much, but am happy to start.

Reply:
No, I thought the entry was in an already-released section, but I maybe was mistaken; sorry.